src/pybind/mgr/dashboard/frontend/src/styles/_carbon-defaults.scss

```diff
@@ -149,3 +150,54 @@ Code snippet
 .cds--snippet {
   width: fit-content;
 }
+
+.cds-success-color {
+  fill: theme.$support-success;
+}
+
+.cds-danger-color {
+  fill: theme.$support-error;
+}
+
+.cds-info-color {
+  fill: theme.$support-info;
+}
+
+.cds-warning-color {
+  fill: theme.$support-warning;
+}
+
+.cds--btn__icon {
+  vertical-align: text-bottom !important;
```

Comment:
Why important here ?

Reply:
its was not aligning with the text so needs to be added